subsys/net/lib/lwm2m/lwm2m_senml_cbor_decode.c

```diff
@@ -81,21 +91,21 @@ static bool decode_value(zcbor_state_t *state, struct value_ *result)
 	bool int_res;
 
 	bool tmp_result =
-		(((zcbor_union_start_code(state) && (int_res = (
-			(((zcbor_tstr_decode(state, (&(*result)._value_tstr)))) &&
+		(((zcbor_union_start_code(state) &&
+		   (int_res = ((((zcbor_tstr_decode(state, (&(*result)._value_tstr)))) &&
 				(((*result)._value_choice = _value_tstr) || 1)) ||
-			(((zcbor_bstr_decode(state, (&(*result)._value_bstr)))) &&
+			       (((zcbor_bstr_decode(state, (&(*result)._value_bstr)))) &&
 				(((*result)._value_choice = _value_bstr) || 1)) ||
-			(((zcbor_int64_decode(state, (&(*result)._value_int))) &&
-				((((*result)._value_int >= INT64_MIN) &&
-				((*result)._value_int <= INT64_MAX)) ||
-				(zcbor_error(state, ZCBOR_ERR_WRONG_RANGE), false))) &&
+			       (((zcbor_int64_decode(state, (&(*result)._value_int))) &&
+				 ((((*result)._value_int >= -9223372036854775807LL) &&
+				   ((*result)._value_int <= 9223372036854775807LL)) ||
```

Comment:
Hmm, to be restored back to its original glory.

Reply:
Aahh missed one, thanks :D